app/services/stockit/item_detail_sync.rb

```diff
@@ -34,7 +34,17 @@ def update
     private
 
     def detail_params
-      params = detail.attributes.except("id", "created_at", "updated_at", "stockit_id", "updated_by_id")
+      params = detail.attributes
+      if (detail_type == "computer") || (detail_type == "computer_accessories")
```

Comment:
['computer',  "computer_accessories"].include?(detail_type)

How about this? 

Just a thought.